frappe/integrations/doctype/braintree_settings/braintree_settings.py

```diff
@@ -0,0 +1,136 @@
+# -*- coding: utf-8 -*-
```

Comment:
lets please use tabs, since the project uses tabs!

Reply:
Hi Rushabh, Sorry about that I should be more careful.
I have modified the PR accordingly.

Thanks and have a nice day!